Misc/NEWS.d/next/Documentation/2019-05-18-16-25-44.bpo-36960.xEKHXj.rst

```diff
@@ -0,0 +1,2 @@
+Restructured the :mod:`datetime` docs in the interest of making them more
+user-friendly and improving readability.
```

Comment:
```suggestion
user-friendly and improving readability. Patch by Brad Solomon.
```

I think you should definitely be credited with this, but it's up to you. I am guessing what your full name is based on your username, but feel free to change this to whatever you prefer.

Reply:
Good by me.

Reply:
@bsolomon1124 Good as in you don't need the patch applied? You marked this resolved but it looks like you didn't accept the suggestion.

Reply:
My fault, just added that suggestion.